### PR TITLE
Create man page for pam_stress

### PR DESCRIPTION
**Pull request by oxr463**
Bug: https://github.com/linux-pam/linux-pam/issues/148

### CODE REVIEW

**Comment by ldv-alt** [comment]
On Sat, Mar 21, 2020 at 04:33:11AM -0700, Lucas Ramage wrote:
> @oxr463 pushed 1 commit.
> 
> e0299c08989fd37562368dbc179eea4846395878  Create pam_stress.8.xml

Looks like there is too much cut-n-paste in this pam_stress.8.xml:

$ git grep pam_access modules/pam_stress
modules/pam_stress/pam_stress.8.xml:      The pam_access PAM module is mainly for access management.
modules/pam_stress/pam_stress.8.xml:  <refsect1 id="pam_access-options">
modules/pam_stress/pam_stress.8.xml:            pam_access will recognize when parsing the access
modules/pam_stress/pam_stress.8.xml:            pam_access with X based applications, since the
modules/pam_stress/pam_stress.8.xml:            pam_access will recognize when parsing the access

**Comment by oxr463** [comment]
@ldv-alt it is not yet ready for review. I copied it from pam_access and I am working through each line. Once I am done, I will remove the WIP prefix from the issue title and request a review.

**Comment by oxr463** [comment]
- [ ] Examples
- [ ] Return Types

**Comment by oxr463** [comment]
@ldv-alt please review once the CI process has completed.

**Comment by oxr463** [comment]
@miztake, @ldv-alt please review the latest changes.

File: modules/pam_stress/pam_stress.8.xml [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+          <option>stress_new_pwd</option>
+        </term>
+        <listitem>
+          <para>
+            Tells pam_sm_chauthtok that
+            pam_sm_acct_mgmt says we need a new
+            password
+          </para>
+        </listitem>
+      </varlistentry>
```
Comment (ldv-alt) [inline modules/pam_stress/pam_stress.8.xml:203]: pam_stress doesn't have stress_new_pwd option.

Comment (oxr463) [reply modules/pam_stress/pam_stress.8.xml:203]: Source: https://github.com/linux-pam/linux-pam/blob/master/modules/pam_stress/README#L62

**Comment by ldv-alt** [comment]
On Tue, Jun 30, 2020 at 04:32:08PM -0700, Lucas Ramage wrote:
> @oxr463 commented on this pull request.
> 
> > +      <varlistentry>
> +        <term>
> +          <option>stress_new_pwd</option>
> +        </term>
> +        <listitem>
> +          <para>
> +            Tells pam_sm_chauthtok that
> +            pam_sm_acct_mgmt says we need a new
> +            password
> +          </para>
> +        </listitem>
> +      </varlistentry>
> 
> Source: https://github.com/linux-pam/linux-pam/blob/master/modules/pam_stress/README#L62

This is not an option of the module, it's a data string used by the module
as a way of communication between its pam_sm_acct_mgmt and pam_sm_chauthtok
functions.

**Comment by oxr463** [comment]
> On Tue, Jun 30, 2020 at 04:32:08PM -0700, Lucas Ramage wrote: @oxr463 commented on this pull request. > + <varlistentry> + <term> + <option>stress_new_pwd</option> + </term> + <listitem> + <para> + Tells pam_sm_chauthtok that + pam_sm_acct_mgmt says we need a new + password + </para> + </listitem> + </varlistentry> Source: https://github.com/linux-pam/linux-pam/blob/master/modules/pam_stress/README#L62
> This is not an option of the module, it's a data string used by the module as a way of communication between its pam_sm_acct_mgmt and pam_sm_chauthtok functions.

How should this be handled in the man page?

**Comment by ldv-alt** [comment]
On Tue, Jun 30, 2020 at 05:16:06PM -0700, Lucas Ramage wrote:
> How should this be handled in the man page?

I suppose it should go to NOTES section.

**Comment by miztake** [comment]
> @miztake, @ldv-alt please review the latest changes.

Sorry for the late reply.
Thank you for correcting it.
I have confirmed that everything I pointed out has been corrected.

**Comment by oxr463** [comment]
@ldv-alt please review the latest changes.

**Pull request closed without merging**